.github/workflows/blackbox-main.yml

```diff
@@ -1,6 +1,9 @@
 name: Blackbox Tests
 
 on:
+  pull_request: # Remove block before marking PR as ready for review
```

Comment:
please remember to remove this block.

Reply:
@licitdev Is this still TBD?

Reply:
@rijkvanzanten It's ready for a merge. The PR should merge automatically once this conversation is resolved.